src/dataset-config.js

```diff
@@ -0,0 +1,518 @@
+'use strict';
+
+const DATASET_ATTRIBUTIONS = {
+    acs: {
+        name: 'U.S. Census American Community Survey',
+        url: 'https://www.census.gov/programs-surveys/acs/'
+    },
+    bea: {
+        name: 'U.S. Bureau of Economic Analysis',
+        url: 'http://www.bea.gov/'
+    },
+    hud: {
+        name: 'U.S. Department of Housing and Urban Development',
+        url: 'http://www.hud.gov/'
+    },
+    rwjf: {
+        name: 'Robert Wood Johnson Foundation',
+        url: 'http://www.rwjf.org/'
+    },
+    cdc: {
+        name: 'Centers for Disease Control and Prevention',
+        url: 'http://www.cdc.gov/'
+    },
+    crimeReports: {
+        name: 'FBI Uniform Crime Reporting Program',
+        url: 'https://www.fbi.gov/about-us/cjis/ucr/ucr'
+
+    },
+    iesNces: {
+        name: 'Institute of Education Sciences at National Center for Education Statistics',
+        url: 'http://nces.ed.gov/ccd/elsi'
+
+    },
+    hhs: {
+        name: 'U.S. Department of Health and Human Services',
+        url: 'http://www.hhs.gov/'
+    }
+};
+
+const DATASET_CONFIG = {
+
+    'crime.fbi_ucr' : {
+        attribution: [DATASET_ATTRIBUTIONS.crimeReports],
+        charts: [
+            {
+                chartId: 'crime.fbi_ucr.count.chart',
+                chartType: 'line',
+                name: 'Crime Incident Count',
+                options: {
+                    hAxis: { format:'####' },
+                    height: 300,
+                    title: 'Crime Incident Count',
+                    vAxis: { format: '###.#' }
+                },
+                variables: [
+                    {
+                        variableId: 'crime.fbi_ucr.count',
+                        label: 'Crime Incident Count',
```

Comment:
Should not need this label. Each variable has a human-readable `name` property. Feel free to tweak those in the backend - they could use some work: https://github.com/socrata/odn-backend/blob/master/data/sources.json#L11